tutorials/continuous-deployment-pipeline-google-container-engine-with-codeship/index.md

```diff
@@ -0,0 +1,275 @@
+
+### Step 5: Encrypt environment variables
+
+You will now encrypt the `.env` file using the Codeship AES key provided in the
+project.
+
+Follow these steps to create an encrypted environment file:
+
+1.  Navigate to the new project's `General` page
+1.  Scroll down to find the `AES Key` header
+1.  Click `Download Key`
+1.  Move the downloaded file to the `hello-express` source code root folder
+1.  Rename this file to `codeship.aes`
+
+Once you have `codeship.aes` in the `hello-express` source code root folder, you
+can run the encrypt command:
+
+    jet encrypt .env encrypted.env
+
+After the `.env` file has been encrypted, you need to update the
+`codeship-services.yml` file to use the encrypted file:
+
+    app:
+      build:
+        dockerfile: Dockerfile
+        image: gcr.io/<<YOUR PROJECT NAME>>/hello-express
+    codeship_gcr_dockercfg:
+      image: codeship/gcr-dockercfg-generator
+      encrypted_env_file: encrypted.env
+      add_docker: true
+    codeship_gce_service:
+      image: codeship/google-cloud-deployment
+      encrypted_env_file: encrypted.env
+      add_docker: true
+      working_dir: /deploy
+      volumes:
+        - ./:/deploy
+
+### Step 6: Commit code to run Codeship build
+
+Codeship will trigger a new build when you push a commit to the remote
+repository. Before you commit the changes, ensure the following files are
+located in `.gitignore`:
+
+    codeship.aes
+    .env
+    /path/to/your/service-account.json
+
+These three files should not be included in your remote repository, as they
+contain sensitive data. Be sure they are excluded from any commits.
+
+You also need to update your `codeship-steps.yml` file to use the
+`gke-update-services` step.
+
+    ...
+    #- name: gke-initial-deployment
+    #  service: codeship_gce_service
+    #  tag: master
+    #  command: bin/deploy --create
+    - name: gke-update-services
+      service: codeship_gce_service
+      tag: master
+      command: bin/deploy
+    # - name: gke-remove-services
+    #   service: codeship_gce_service
+    #   command: bin/deploy --remove
+
+Once you have saved these changes, stage all of these files for a commit. Commit
+the changes to the master branch, and push to your remote repository. After the
+push is complete, return to the Codeship project dashboard and watch for a green
+build.
+
+### Step 7: Deploy an update to your application (optional)
+
+You can now modify the code in the `server.js` file to say something different.
+Change `'Hello Express!'` to something like `'Hello World!'`. Once you have
+changed the application, commit the change and push to your remote repository.
+
+After the Codeship process has completes, navigate again to the application
+endpoint to verify your changes have taken effect.
+
+## Cleaning up
+
+After completing the tutorial, follow these steps to remove the resources in
+Google Cloud to prevent any charges:
```

Comment:
s/Cloud/Cloud Platform/

Reply:
or: `in your Google Cloud Platform Account to prevent...`

Reply:
I went with `remove the resources from your Google Cloud Platform account`